Author: Gregor Riepl <onitake@gmail.com>
Description: Fonts should be packaged separately.
  This patch excludes the installation of Open Sans,
  so a system version is picked up instead.
  A dependency on fonts-open-sans was added to debian/control.
Index: Cura/CMakeLists.txt
===================================================================
--- Cura.orig/CMakeLists.txt	2020-04-14 15:54:27.921480550 -0600
+++ Cura/CMakeLists.txt	2020-04-14 15:54:27.921480550 -0600
@@ -37,7 +37,8 @@
 find_package(PythonInterp 3.5.0 REQUIRED)
 
 install(DIRECTORY resources
-        DESTINATION ${CMAKE_INSTALL_DATADIR}/cura)
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/cura
+        PATTERN resources/themes/cura-light/fonts EXCLUDE)
 install(DIRECTORY plugins
         DESTINATION lib${LIB_SUFFIX}/cura)
 if(NOT APPLE AND NOT WIN32)
